examples/testsuites/helloworld.py

```diff
@@ -17,6 +18,7 @@
     this is an example test suite.
     it helps to understand how to write a test case.
     """,
+    requirement=simple_requirement(unsupported_os=[]),
```

Comment:
Do we want some OS not to have support to even run basic LISA test?

Reply:
So far, there is no such OS discovered. The hello world examples are very simple, it should be able to run in any situation. If they cannot be run, even the OS detection doesn't work.

Reply:
Then what's the point of having this list if we don't have any such OS yet.

Reply:
Because the default value is [Windows] now, if we don't set it's an empty list.